jsonschema/src/keywords/pattern.rs

```diff
@@ -110,14 +117,16 @@ fn convert_regex(pattern: &str) -> Result<Regex, regex::Error> {
             out.push(current);
         }
     }
-    Regex::new(&out)
+    fancy_regex::Regex::new(&out)
 }
 
 #[allow(clippy::integer_arithmetic)]
-fn replace_control_group(captures: &Captures) -> String {
+fn replace_control_group(captures: &regex::Captures) -> String {
     // There will be no overflow, because the minimum value is 65 (char 'A')
     ((captures
-        .index(0)
+        .get(0)
+        .map(|m| m.as_str())
+        .expect("index 0 to return the whole match")
```

Comment:
As the type didn't change, why are these changes needed?

Reply:
Good catch. This got left over from my initial attempt to use `fancy_regex::Captures` here. Now that we're continuing to use `regex` this is no longer necessary, so I'll revert it.